File: Python/2968.py
cont=1
i=1
linha=input().split(" ")
voltas=int(linha[0])
placas=int(linha[1])
total=voltas*placas
i=total//10
while cont!=10:
    if (i>=(cont*total)/10):
       if(cont!=10): print(i, end="")
       else: print(i)
       cont+=1
       i+=total//10
       if (cont!=10):
           print(" ",end="")
    else:
        i+=1

File: C/2968.c
#include <stdio.h>
int main() {
    double cont=1.0, i=1.0, voltas, placas, total;

    scanf("%lf %lf", &voltas, &placas);
    total=voltas*placas;
    while (cont!=10.0){
        if (i>=(cont*total)/10.0){
            printf("%.0lf", i);
            if (cont!=9) printf(" ");
            else printf("\n");
            cont++;
            i++;
        }
        else i++;
    }
    
    return 0;
}
